manifest.yaml

```diff
@@ -249,3 +249,22 @@ backup:
       main: /root/.lnd
       compat: /mnt/assets
     io-format: yaml
+migrations:
+  from:
+    "<=0.14.2.1":
```

Comment:
probably easier to maintain if you change this to `<0.14.2.2`, but will work this way too